src/combo_box.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ComboBox;
+use ffi;
+use glib::object::IsA;
+
+pub trait ComboBoxExtManual {
+    fn set_active<'a, P: Into<Option<u32>>>(&self, index_: P);
```

Comment:
`'a` is really needed ?

Reply:
No, and I'm surprised it's not a compiler warning :)